keyboards/keychron/c2_pro/ansi/rgb/rules.mk

```diff
@@ -0,0 +1,13 @@
+# Build Options
+#   change yes to no to disable.
+#
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = embedded_flash
```

Comment:
```suggestion
```

This is already set as the default for the STM32L4 controllers, and does not need to be configured here. 

https://github.com/qmk/qmk_firmware/blob/a4b36f0b4b45a09ab940910b11b9be52356c721a/builddefs/common_features.mk#L236-L240